components/api_version/src/lib.rs

```diff
@@ -261,11 +261,21 @@ mod tests {
         assert_eq!(ApiV1::parse_key_mode(&b"t_a"[..]), KeyMode::Unknown);
         assert_eq!(ApiV1Ttl::parse_key_mode(&b"ot"[..]), KeyMode::Raw);
         assert_eq!(
-            ApiV2::parse_key_mode(&[RAW_KEY_PREFIX, b'a', b'b']),
+            ApiV2::parse_key_mode(&[RAW_KEY_PREFIX, 0, 0, 0, b'a', b'b']),
```

Comment:
Suggest use ```DEFAULT_KEY_SPACE_ID``` instead of ```0, 0, 0```

Reply:
Maybe I should expand the slice to `r\x00\x00\x00ab`.